src/main/java/ru/hse/software/design/CLI.java

```diff
@@ -0,0 +1,68 @@
+package ru.hse.software.design;
+
+import java.util.Scanner;
+
+/**
+ * Contains custom realization of command-line interpreter for bash commands.
+ * Details about accepted commands are described in README.md.
+ */
+
+public class CLI {
```

Comment:
```suggestion
 */
public class CLI {
```
Пустая строка обычно не ставится

Reply:
Fixed